src/main/java/ua/hillel/moneymanager/repository/ExpenditureRepository.java

```diff
@@ -0,0 +1,20 @@
+package ua.hillel.moneymanager.repository;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import ua.hillel.moneymanager.dto.ExpenditureDto;
+import ua.hillel.moneymanager.entity.Category;
+import ua.hillel.moneymanager.entity.Expenditure;
+import ua.hillel.moneymanager.service.expenditure.ExpenditureService;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.List;
+
+public interface ExpenditureRepository extends JpaRepository<Expenditure, Long> {
+
+    void deleteById(Long id);
```

Comment:
Spring data should provide such method out of the box, no need to declare it explicitly